### PR TITLE
completed 1st SPARQL guide

### PR DESCRIPTION
**Pull request by charlesivie**
Issue #, if available:

Description of changes:

The first of some SPARQL tutorials to be included in the notebooks. This is the basics. 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb [hunk 1]
```diff
@@ -0,0 +1,1412 @@
...
+   },
+   "cell_type": "markdown",
+   "id": "94a47c81",
+   "metadata": {},
+   "source": [
+    "# Using the SPARQL1.1 query language\n",
+    "\n",
+    "Working with Semantic Web knowledge graphs can be a daunting prospect for a newcomer.  Having lots of new acronyms and terms to learn and understand – like RDF, SPARQL and knowledge graph – can been seen as a barrier of entry for these powerful technologies. \n",
+    "\n",
+    "In reality, Semantic Web knowledge graph databases are a very easy and intuitive way to understand, record and traverse data. In our opinion, much easier to use than relational databases. \n",
```
Comment (krlawrence) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:17]: I suggest rewording this a little from "In our opinion...." to something like "This is often easier to do using graph databases, and graph query languages, as they are optimized for these types of operation"

File: src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb [hunk 2]
```diff
@@ -0,0 +1,1412 @@
...
+    "\n",
+    "## Prerequisites\n",
+    "\n",
+    "This guide will show you how easy it is to use the powerful features of RDF and SPARQL, and as long as you have a general understanding of how to use a REST API, and some experience with some other kind of database in the past, SQL or NoSQL, you will be able to follow along easily.\n",
```
Comment (krlawrence) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:21]: This guide shows

**Review by krlawrence — CHANGES_REQUESTED** [review]
I need to finish reading but suggested a few upfront wording changes

File: src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb [hunk 3]
```diff
@@ -0,0 +1,1412 @@
...
...u+22bVww7j+T/Jn8xQmKAAECBAjkXKCrdxSmJj3//PPF15LkvHnCbyfQcaK33a7y1xEjRrQV/k6IhRdeuLyt45eO1ym9sqfjcWn9L3/5S/E1Pu33tS88SUUvP/jBD9rvjg8//LDLJ62kgx599NFYddVVK46/7rrrYtNNNy1uS5833HBDeX8qCBk3blxxffvtty++OiitpOKa9Jjqb33rW+VjU6ylIpiVV145nnjiifK+O+64I9ZZZ53ieophtdVW6/REoNLTUMaOHVt8WkpXT4spX9CXTAvITTLdPYIjQIAAgSYWkJs0cee2Na1jHtFVa+uZj1Q7F0ntqXU+Ihfp6lfT3NvkJs3dv1pHgAABAtkVaNXcpCkKT9LPqrvq3QsuOCXWH7lWdn95IiNAgAABAjkVeGHi5Bg6dO1O0XvaSSeSptgwq4neT33qU/HUU0/FwIEDZ9rejtfpS+FJKjpp/8SVRRddNF5//fVu7//GG2/EYostVrH/tNNOi/SEkrT85je/iT322KO8f7nlliu+Iie9Nir9bcrXXnutuO/II4+M/fffPxZffPGYMWNGcVsqRDnppJPiX//6V/lVQGnH0ksvHZMnTy6+Zqd4YNv/XHrppW0F0rsVi2RK29p/Dho0KA455JDYe++9uy2iaX+879kTkJtkr09ERIAAAQLNLSA3ae7+Ta3rmEd0bHG985Fq5yKpPfXIR+QiHX85zb8uN2n+PtZCAgQIEMiWQCvnJk1TeNJd9e7IkWvGhFv/mK1fnGgIECBAgEATCKw/aru47bZ7OrUk/SG9pfkEZjXRm57YccEFF5RfTdOdQMfr9KXwZM8994xzzjmnfKsll1wyXnrppfJ6xy/vvfde+XU4pX2pWCQVjaRlypQpsdRSS8X06dNLuyO9u/3tt9+OL3zhC+Vt6akuI0eOjC996Utx//33F7en/X//+98jvZ4nPSmltKSillTc0nFJrwRKxSdPPvlkx13l9TXXXDOuueaaTsUy5QN8yayA3CSzXSMwAgQIEGhSAblJk3Zsu2Z1zCPa7Sp+rXc+Uu1cJDWiXvmIXKTjr6e51+Umzd2/WkeAAAEC2RNo5dykaQpP0s8q/WHHmDFjOv3Cjjpq/zj6qAM6bbeBAAECBAgQ6J1Ad4+KS68laf+H7r27urOyKNBxojc97WPq1Kkxbdq0crhzzTVXXHHFFbHF
```
Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:464]: Would be nice to have a graphic of this too

Comment (joywa) [reply src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:464]: I think this data is a bit hard to relate to for a beginner who doesn't know about "TheSemanticWeb" and the related books.

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:490]: OPTIONAL clause

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:573]: Not sure why it shows 2. on here, but on the preview in your branch, it shows "1. Only return Authors who have published something..." Can you verify?

https://github.com/aws/graph-notebook/blob/SPARQL-TUTORIAL/src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:589]: Nit: We haven't talked about periods in these SPARQL queries. Are they necessary? Why do some lines have them and others not? Is the spacing before a period important at all?

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:705]: Using GROUP BY clause and filters

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:775]: I think we should show this example first to demonstrate the benefit provided by GROUP BY. Also numbering is off again in the actual notebook.

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:740]: Is it ok for this GROUP BY to not be capitalized?

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:1047]: Using LIMIT clause

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:1197]: Using the ORDER BY clause

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:1201]: "alphabetically"

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:1256]: Using DISTINCT clause

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:1337]: Using COUNT _____. Is this a clause, filter, function, or something else?

Comment (joywa) [inline src/graph_notebook/notebooks/06-Language-Tutorials/01-SPARQL/01-SPARQL-BASICS.ipynb:1390]: We should add a Conclusion section with calls-to-action. E.g., linking to additional notebooks, link to sample tutorials, link to console to create a new cluster and notebook today.. Check out the Sample Applications notebook conclusions for ideas.

**Comment by joywa** [comment]
Worked with @charlesivie offline, latest draft looks good for merging.

**Review by krlawrence — DISMISSED** [review]

**Review by michaelnchin — APPROVED** [review]
